client/src/components/new-request/quick-search-results.vue

```diff
@@ -96,9 +101,11 @@ export default class QuickSearchResults extends Vue {
     newReqModule.mutateQuickSearch(true)
     newReqModule.startAnalyzeName()
   }
-  detailedSearch () {
+  async detailedSearch () {
+    this.isLoadingAnalysis = true
     newReqModule.mutateQuickSearch(false)
-    newReqModule.startAnalyzeName()
+    await newReqModule.startAnalyzeName()
```

Comment:
There are many other places in the code where startAnalyzeName is called. Do you think it's worth changing those to awaits also?

Reply:
I'm not sure.. 
I don't see a downside to changing, but i'm not sure it will have any unexpected implications. This would require further testing on each call i think.

Reply:
Updated!